scripts/portfolioView.js

```diff
@@ -0,0 +1,91 @@
+(function(module){
+  var portfolioView = {};
+  var $headerPosition = $('header').offset().top;
+
+  // helper functions
+  // toggle sticky header class based on scroll position
+  function stickyHeader(){
+    if($(this).scrollTop() > $headerPosition){
+      $('header').addClass('sticky');
+    } else {
+      $('header').removeClass('sticky');
+    }
+  };
+
+  // on click build detailed view of project
+  portfolioView.detailHandler = function (){
+    $('#portfolio').on('click','.project-preview',function(){
+      $('.project-details').insertAfter($(this));
+      var $clickedItem = $(this).data('project');
+      PortfolioItem.all.forEach(function(a){
+        if(a.project === $clickedItem){
+          $('.project-details').html(a.updateDetailModal());
+          setTimeout(function(){
```

Comment:
Investigate refactoring. There must be a way for this to live as a callback function after .insertAfter occurs. As of now without the delay this won't fire as insertAfter must interfere with delegation.